blog/_posts/TeamDelSquared.md

```diff
@@ -0,0 +1,40 @@
+Worked on an ESP32 Hello World tutorial for Windows 10 and submitted a blogpost for the same. (currently under review)
+Had a mini Tessel hackathon in which we tried to build a SoundBot of sorts. We controlled LED bulbs with live music and made them fluctuate in sync with the music beats. We used a Tessel board and an ambient module for the same. The blogpost is currently under review and would be out soon!
+Figured out a way to build diagrams of the Tessel components using Fritzing and submit them in the required format. We really had to bang our hands for this one. It was proving to be pretty annoying :p 
+Worked on a few documentation issues, some of which we were able to close while others are under review or need to be updated.
+Learnt a lot about the I2C protocol as we had to browse a lot of online resources to learn how devices talk to each other using these communication protocols. We are currently working on implementing I2C to create a demo for communication between an ESP32 and an accelerometer module.
+Worked with several APIs and softwares like Moddable, ImageMagick. We’re still exploring Moddable but it seems like a amazing tool for communicating to micro-controllers which are based on JavaScript.
+
+ **Looking Ahead** </br>
+We intend to keep working on a few aspects of Tessel Reach and the Fritzing issues because we are pretty close to solving them. Also, we found a lot of interesting resources and tools that we can share with the people who are interested in working with Tessel. 
+Some of the resources we found were: </br>
+Moddable API [https://github.com/Moddable-OpenSource/moddable] </br>
+Working with Fritzing [https://learn.adafruit.com/make-beautiful-fritzing-parts-with-eagle2fritzing-brd2svg/get-ready-git-clone] </br>
+As and when time permits, we are thinking of writing a blog detailing a few approaches we have in mind, which future contributors to Tessel can use and achieve. We weren’t able to fulfil these in the limited time we had, but we would surely like to see these tried out or try out ourselves sometime we’re free.
+This summer of code has had a profound impact on us, as we have learned not just the technical stuff but also working on real-life projects and working with people across the globe. Sitting down to summarise our work every week, and setting targets for the next week helped us and the mentors evaluate our progress, and we have progressively worked on the feedback we received each week, bettering ourselves. We have also started embracing unfamiliarity and challenges, as we learnt about many new things we had never heard of before, like Moddable. From being novices at understanding the technical intricacies, we have come a far way and we believe this knowledge will immensely benefit us, as we both want to pursue a career in robotics.
+
+Last but not the least…(*drumrolls*) </br>
+**A huge shout out to our amazing team!** </br>
```

Comment:
Same comment as above. 
- Add blank lines in between the paragraphs to make the text readable. 
- Check if all br tags are giving the output as expected. If not then remove them. 

Also, it would be good if you can fit any image somewhere in the blog post. :)